lms/templates/registration/activation_complete.html

```diff
@@ -23,9 +25,11 @@ <h1 class="valid">${_("Activation Complete!")}</h1>
       %endif
 
       %if user_logged_in:
-        ${_("Visit your {link_start}dashboard{link_end} to see your courses.").format(link_start='<a href="{url}">'.format(url=reverse('dashboard')), link_end='</a>')}
+        ${Text(_("Visit your {link_start}dashboard{link_end} to see your courses.")).format(
+            link_start=HTML('<a href="{url}">)'.format(url=reverse('dashboard')), link_end=HTML('</a>'))}
       %else:
-        ${_("You can now {link_start}log in{link_end}.").format(link_start='<a href="{url}">'.format(url=reverse('signin_user')), link_end='</a>')}
+        ${Text(_("You can now {link_start}log in{link_end}.")).format(
+            link_start=HTML('<a href="{url}">)'.format(url=reverse('signin_user')), link_end=HTML('</a>'))}
```

Comment:
Same thing: the ending single quote for link_start seems to be in the wrong place.